### PR TITLE
Add Turing

### PR DESCRIPTION
**Pull request by imstar15**
Turing's address prefix: 27

### CODE REVIEW

File: ss58-registry.json [hunk 1]
```diff
@@ -253,6 +253,15 @@
       "standardAccount": "*25519",
       "website": "https://jupiter.patract.io"
     },
+    {
+      "prefix": 27,
+      "network": "turing",
+      "displayName": "Turing",
```
Comment (chrisli30) [inline ss58-registry.json]: Turing Network

File: ss58-registry.json [hunk 2]
```diff
@@ -253,6 +253,15 @@
...
+      "symbols": ["TUR"],
+      "decimals": [10],
+      "standardAccount": "*25519",
+      "website": "https://oak.tech"
```
Comment (chrisli30) [inline ss58-registry.json]: https://oak.tech/turing

**Review by chrisli30 — CHANGES_REQUESTED** [review]
Let’s only PR this to upstream when `https://oak.tech/turing` page is available in 2 weeks.

**Review by chrisli30 — APPROVED** [review]
Remember to merge it to Paritytech when "https://oak.tech/turing" is up.

**Comment by irsal** [comment]
> Remember to merge it to Paritytech when "https://oak.tech/turing" is up.

Don't we have to do a PR in the real repo when we're ready? https://github.com/paritytech/ss58-registry

**Comment by chrisli30** [comment]
> > Remember to merge it to Paritytech when "https://oak.tech/turing" is up.
> 
> Don't we have to do a PR in the real repo when we're ready? https://github.com/paritytech/ss58-registry

Correct, that’s exactly what I meant, merging the change to the upstream.

**Pull request closed without merging**